sktime/forecasting/compose/tests/test_reduce_global.py

```diff
@@ -358,3 +358,30 @@ def test_nofreq_pass():
     np.testing.assert_almost_equal(
         y_pred_global["c0"].values, y_pred_nofreq["c0"].values
     )
+
+
+def test_timezoneaware_index():
+    y = load_solar(api_version=None)
+    y_notz = y.copy().tz_localize(None)
+
+    window_trafo = WindowSummarizer(n_jobs=1, **{"lag_feature": {"lag": [1, 2, 48]}})
+    regressor = LinearRegression()
+    forecaster = make_reduction(
+        estimator=regressor,
+        strategy="recursive",
+        transformers=[window_trafo],
+        window_length=None,
+        pooling="global",
+    )
+
+    # check coefficients
+    tzaware = forecaster.clone().fit(y).get_fitted_params()["estimator__coef"]
+    tznaive = forecaster.clone().fit(y_notz).get_fitted_params()["estimator__coef"]
+    np.testing.assert_almost_equal(tzaware, tznaive)
+
+    fh = np.arange(1, 97)
+    pred_tzaware = forecaster.clone().fit_predict(y, fh=fh)
+    pred_tznaive = forecaster.clone().fit_predict(y_notz, fh=fh)
```

Comment:
Non-blocking suggestions: `assert pred_tzaware.index.tz is not None` and `assert pred_tznaive.index.tz is None`.

Reply:
just checked here the time zone was consistent with the input data